src/sass/components/_contacts.scss

```diff
@@ -0,0 +1,136 @@
+.contacts{
+    background-color: $pageBg;
+    display:flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items:center;
+    padding:86px 50px 113px;
```

Comment:
jak już opisujesz padding dla wszystkich stron, to musisz określić wszystkie 4 strony, a msz tylko 3 wartości. wpisz 0 po prostu :)